helix-term/src/main.rs

```diff
@@ -91,8 +92,39 @@ FLAGS:
 
     setup_logging(logpath, args.verbosity).context("failed to initialize logging")?;
 
+    let themes = std::sync::Arc::new(Themes::load());
+
+    // initialize language registry
+    use helix_core::syntax;
+
+    // load $HOME/.config/helix/languages.toml, fallback to default config
+    let lang_config = std::fs::read(conf_dir.join("languages.toml"));
```

Comment:
I think all this was recently moved into `Application::new`, can we just do it there?